### ISSUE TITLE
[BUG] Comp breaks it / add relaunch option?

### ISSUE THREAD

**Issue by LoonEc** [PazerOP/tf2_bot_detector#369]
The program stops working, needing to relaunch TF2

If you join a comp match, the game breaks (as expected, since you can't kick people on comp). It makes text from the console appear on the top left corner too.
The problem is that it doesn't start working again once you join back to casual and you need to relaunch everything from zero.

Maybe a small button tho tell the program to reconnect would work?

**Comment by UAVXP** [comment]
Yeah, I've noticed that the game, after launching competitive match, cannot change some convars, like "contimes" and "con_filter_enable"

**Comment by ClusterConsultant** [comment]
Valve comp breaks a lot of things and we cannot really control that.

**Comment by LoonEc** [comment]
I cannot confirm this rn, but it happens in any tournament mode.
I remember it also broke with face.it I'd tried it, but apparently face.it died as soon as I blinked, and I am not part of other tournament thing

**Comment by ClusterConsultant** [comment]
Faceit broke due to their anticheat being bad. There isn't really support in general for community servers, but I have not had any issue on any league config to date.
